server/utils/bookings/index.ts

```diff
@@ -21,9 +21,14 @@ import { bookingActions, v1BookingActions, v2BookingActions } from './bookingAct
 
 const UPCOMING_WINDOW_IN_DAYS = 365 * 10
 
+enum ConflictingEntityType {
+  booking,
+  lostBed,
+}
+
```

Comment:
I don't think the codebase uses enums anywhere, because 'enums are evil' or whatever -- not sure I want to go down this debate, but, best keep to union types, if anything, for consistency.